### PR TITLE
add more detections for bots

### PR DESCRIPTION
**Pull request by jurahudak**
While setting up a piwik instance I came across several bots in our access log.

In this case the bots had their "origin" url written in USER_AGENT. I have another dozen without proper naming or url. If interested I can create a pull request even with these.

### CODE REVIEW

File: regexes/bots.yml [hunk 1]
```diff
@@ -349,6 +362,14 @@
     name: 'Genieo'
     url: 'http://www.genieo.com'
 
+- regex: 'Gluten Free Crawler'
+  name: 'Gluten Free Crawler'
+  category: 'crawler'
+  url: 'http://glutenfreepleasure.com/'
+
+- regex: 'Guzzle'
+  url: 'https://github.com/guzzle/guzzle'
```
Comment (sgiehl) [inline regexes/bots.yml]: a bot definition should include a name

**Comment by sgiehl** [comment]
Thanks for the PR. would you mind adding some tests for the new bots or providing a list with example user agents so I could add them after merging?

**Comment by jurahudak** [comment]
Well, if it's putting them into Test/Parser/fixtures/bots.yml I can give it a try.

Nevertheless, there are the user agent strings:

Blogtrottr/2.0
BUbiNG (+http://law.di.unimi.it/BUbiNG.html)
Domain Re-Animator Bot (http://domainreanimator.com) - support@domainreanimator.com
Mozilla/5.0 (compatible; Gluten Free Crawler/1.0; +http://glutenfreepleasure.com/)
Guzzle/3.9.3 curl/7.38.0 PHP/5.6.14-0+deb8u1
ICC-Crawler/2.0 (Mozilla-compatible; ; http://www.nict.go.jp/en/univ-com/plan/crawl.html)
larbin_2.6.3 larbin2.6.3@unspecified.mail
ltx71 - (http://ltx71.com/)

ROI Hunter; https://api-dev.roihunter.com
ROI Hunter; https://api.roihunter.com

Scrapy/1.0.3 (+http://scrapy.org)
Scrapy/0.24.6 (+http://scrapy.org)
Scrapy/1.0.3.post6+g2d688cd (+http://scrapy.org)
Scrapy/0.22.2 (+http://scrapy.org)

TelegramBot (like TwitterBot)

File: regexes/bots.yml (hunk 1 above)
Comment (jurahudak) [inline regexes/bots.yml]: Ok, did miss that one. Even it's/it'll be made up from the user agent.

What now? Should I add it into the file and make it another commit/request (maybe even with the tests) or will you add it on merge?

**Comment by sgiehl** [comment]
Yes. It's adding the to the fixture. But I'll do that now.
Regarding the other useragents. You could also create an issue with the list of bot user agents and I will consider adding them.
